crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -52,6 +54,24 @@ def knife_files
           file_path = new_file
         end
 
+        if file.basename.to_s =~ /^bc-network-(.*)\.json$/
+          json = JSON.load(file.read)
+          # override with migrated attributes
+          json["attributes"] = SchemaMigration.migrated_attributes("network", json)
```

Comment:
With the suggested changes in ```migrated_attributes``` you need to assign the result to json["attributes"]["network"].